static/app/views/replays/replayTable.tsx

```diff
@@ -199,31 +231,54 @@ function ReplayTableRow({
       {showProjectColumn && minWidthIsSmall && (
         <Item>{project ? <ProjectBadge project={project} avatarSize={16} /> : null}</Item>
       )}
-      <Item>
-        <TimeSinceWrapper>
-          {minWidthIsSmall && <StyledIconCalendarWrapper color="gray500" size="sm" />}
-          <TimeSince date={replay.startedAt} />
-        </TimeSinceWrapper>
-      </Item>
+      {replay.txEvent && minWidthIsSmall ? (
+        <Item>
+          <SpanOperationBreakdown>
+            {txDuration ? <TxDuration>{txDuration}ms</TxDuration> : null}
+            {spanOperationRelativeBreakdownRenderer(replay.txEvent, {
+              organization,
+              location,
+            })}
+          </SpanOperationBreakdown>
+        </Item>
+      ) : null}
+      {minWidthIsSmall && (
+        <Item>
+          <TimeSinceWrapper>
+            {minWidthIsSmall && <StyledIconCalendarWrapper color="gray500" size="sm" />}
+            <TimeSince date={replay.startedAt} />
+          </TimeSinceWrapper>
+        </Item>
+      )}
       <Item>
         <Duration seconds={Math.floor(replay.duration)} exact abbreviation />
       </Item>
       <Item data-test-id="replay-table-count-errors">{replay.countErrors || 0}</Item>
-      <Item>
-        <ReplayHighlight replay={replay} />
-      </Item>
+      {minWidthIsSmall && (
```

Comment:
i don't think that fixing things for mobile viewports is a blocker for launch. The replay details view itself not at all optimized and it's not on the roadmap right now.